jhove-core/src/main/java/edu/harvard/hul/ois/jhove/handler/TextHandler.java

```diff
@@ -1073,19 +1073,23 @@ private void writeAESTimeRange (String baseIndent,
         _writer.println (margn2 + "TimeBase: 1000");
         _writer.println (margn2 + "VideoField: FIELD_1");
         _writer.println (margn2 + "CountingMode: NTSC_NON_DROP_FRAME");
-        _writer.println (margn2 + "Hours: " + Integer.toString (start.getHours ()));
-        _writer.println (margn2 + "Minutes: " + Integer.toString (start.getMinutes ()));
-        _writer.println (margn2 + "Seconds: " + Integer.toString (start.getSeconds ()));
-        _writer.println (margn2 + "Frames: " + Integer.toString (start.getFrames ()));
+        _writer.println (margn2 + "Hours: " +
+                Long.toString (start.getHours ()));
+        _writer.println (margn2 + "Minutes: " +
+                Long.toString (start.getMinutes ()));
+        _writer.println (margn2 + "Seconds: " +
+                Long.toString (start.getSeconds ()));
+        _writer.println (margn2 + "Frames: " +
+                Long.toString (start.getFrames ()));
         _writer.println (margn2 + "Samples: ");
 	double sr = start.getSampleRate ();
 	if (sr == 1.0) {
 	    sr = _sampleRate;
 	}
 	_writer.println (margn3 + "SampleRate: S" +
 			 Integer.toString ((int) sr));
-	_writer.println (margn3 + "NumberOfSamples: " + 
-			 Integer.toString (start.getSamples ()));
+        _writer.println (margn3 + "NumberOfSamples: " +
+                Long.toString (start.getSamples ()));

```

Comment:
Mixed spacing on this and the next four changes from Integer to Long.

Reply:
Yeah, I probably should have matched the spacing with the line directly above instead of the line beneath, but this is one of those areas that is already in a sea of inconsistencies.